kernel.spec.in

```diff
@@ -159,7 +154,7 @@ cd linux-%upstream_version
 # drop EXTRAVERSION - possible -rc suffix already included in %release
 sed -i -e 's/^EXTRAVERSION = -rc.*/EXTRAVERSION =/' Makefile
 
-%if 0%{?fedora} >= 31
+%if 0%{?fedora} >= 31 || 0%{?centos} >= 8
```

Comment:
I think other spec use `%{?rhel}` instead. Any reason for centos here?

Reply:
Yes indeed. Mistaken having Makefile in mind.